.github/ISSUE_TEMPLATE/refactor.yml

```
@@ -13,36 +13,36 @@ body:
 
-  - type: checkbox
+  - type: checkboxes
     attributes:
       label: "Checklist"
       options:
         - "Ensure the refactor does not introduce new bugs"
         - "Update unit tests if applicable"
         - "Ensure the refactor does not negatively impact performance"
         - "Document any new patterns or architecture changes"
-        - "Test the refactor in different environments (e.g., staging, production)"
+        - "Test the refactor in different environments (e.g., staging, production)"
```

yamllint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)